src/components/Heading/Heading.tsx

```diff
@@ -27,13 +28,28 @@ export type HeadingTagTypes = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'span' |
 
 type ElementProps = Omit<HTMLAttributes<HTMLElement>, keyof Props | keyof TextProps>
 
+function generateTagAttribute(level: number): HeadingTagTypes {
+  if (level <= 6) {
+    return `h${level}` as HeadingTagTypes
+  }
+
+  return 'span'
```

Comment:
`tag` は `@deprecated` にしつつ、既存の動きは変えずに無指定は `h1` になる認識です。

```suggestion
  return 'h1'
```

Reply:
このfunctionに入る == tagが未指定の場合 == levelは0となるため、ここでh1にすると色々おかしなことになります。

Reply:
Heading のデフォルトが h1 という、既存の振る舞いを変えずに deprecated する話がリファインメントの段階でありましたが、別の所で担保されてそうですか?（BREAKING CHANGE にはしない話があった記憶）

Reply:
はい、担保されています！
levelは常に数値になっており、SectioningContentでラップされていない箇所では1から始まるのでご指摘の箇所に入るパターンはSectioningContentが6つ以上ネストしている場合だけです。
そうでない箇所では常にh1がデフォルトになります